src/specs/spec-equivalents.json

```diff
@@ -71,10 +71,6 @@
     "https://www.w3.org/TR/html-media-capture/": [
         "https://dev.w3.org/2009/dap/camera/"
     ],
-    "https://www.w3.org/TR/service-workers-1/": [
```

Comment:
The equivalences listed here merely affect the way Reffy will look at references to answer the question: "This spec uses IDL terms defined in Service Workers, does it reference it normatively?" The exact URL used for the reference does not matter so much as the presence of the reference (although we could certainly add a test about URL consistency later on).

In other words, I would still keep these equivalences as is.

(I haven't checked whether these specific equivalences for Service Workers are still useful though).